code/modules/clothing/masks/gasmask.dm

```diff
@@ -19,7 +19,27 @@
 		"Grey" = 'icons/mob/species/grey/mask.dmi',
 		"Plasmaman" = 'icons/mob/species/plasmaman/mask.dmi'
 		)
+var/gasmask_cut = FALSE
```

Comment:
Needs to be indented by one tab so that it's part of `/obj/item/clothing/mask/gas`.
```suggestion
	var/gasmask_cut = FALSE
```